src/SDKs/DataFactory/Management.DataFactory/changelog.md

```diff
@@ -4,12 +4,26 @@
 
 ###  Feature Additions
 
-* Support sharing self-hosted integration runtime across data factories and subscriptions
+* Added support for sharing self-hosted integration runtime across data factories and subscriptions
+* Added support for Databricks Spark Jar and Databricks Spark Python activities
 
-## Version 0.9.0-preview
+## Version 1.0.0
```

Comment:
Shouldn't this be 1.1.0?

Reply:
No this is the previous version that got lost. 1.1.0 is the current version